guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,395 @@ Illustration sources:
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-force-close-attempt@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-force-close-attempt@2x.png"
+   layout = "shadow"
+   caption = "App sends a signal back to the LSP to attempt to force close the channels."
+   alt-text = "Screen showing an animation while the app attempts to restore"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-force-close-success.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-force-close-success@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-force-close-success@2x.png"
+   layout = "shadow"
+   caption = "The force close has been initiated, so the restore has effectively worked."
+   alt-text = "Screen showing that the restore worked."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-pending-notice.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-pending-notice@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-pending-notice@2x.png"
+   layout = "shadow"
+   caption = "App reminds the user of the delay in accessing funds."
+   alt-text = "Screen describing the delay in accessing funds"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-finish-pending.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-finish-pending@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-finish-pending@2x.png"
+   layout = "shadow"
+   caption = "The delay is also indicated on the homescreen of the app."
+   alt-text = "Screen showing the wallet homescreen with pending   indicators"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+### Handling emergency scenarios
+
+In a situation where the app provider or LSP is offline or uncooperative, the user will have no assistance in getting their funds. While this situation is unlikely, it should be prepared for.
+
+This wallet offers an open source wallet recovery tool. This tool is available on GitHub and linked to from their website. In the event that the app is taken down from app stores or the LSP's node is down for an extended period of time, the user can use their backup to force close their channels and have all their bitcoin sent to an on-chain address.
```

Comment:
You could have this in an Emergency section in the settings that simply uses a SCB to force close all channels, a separate tool is only necessary if the user can't even access the app at all which is the most extreme emergency scenario. 

Phoenix has this in their app for an example to look at.

Reply:
There's already a section about force closing within the app itself. Its 3 lines above the section you are commenting on.

![Screen Shot 2022-08-19 at 7 32 38 AM](https://user-images.githubusercontent.com/2414177/185609660-e8c8bb31-8008-40be-ad93-e44da1c1f3f6.png)